.jshintrc

```diff
@@ -1,15 +1,40 @@
 {
-  "camelcase": true,
+  // Put curly braces around conditional blocks & loops
   "curly": true,
+
+  // Use deep-equals for comparison
   "eqeqeq": true,
+
+  // Prohibit extension of native objects, i.e., String
```

Comment:
Could you use "overwriting" instead of "extensive"? It's a bit more straightforward. http://jshint.com/docs/options/#freeze

Reply:
Yes!